UnityProject/Assets/Scripts/Communications/SignalReceiver.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Managers;
+using Mirror;
+using UnityEngine;
+using ScriptableObjects.Communications;
+
+namespace Communications
+{
+	public class SignalReceiver : NetworkBehaviour
+	{
+		public SignalType SignalTypeToReceive = SignalType.PING;
+		[SyncVar] public float Frequency = 122F;
+		[SyncVar] public SignalEmitter Emitter;
```

Comment:
Unlikely you'll need to sync this, also wont work

Reply:
Tbh, doubt any of this will need syncvaring, nettabs will do that

Reply:
will add syncvars later if they seem to be needed.